tailscale/DOCS.md

```diff
@@ -54,6 +54,11 @@ subnets on all supported interfaces to Tailscale.
 Consider disabling key expiry to avoid losing connection to your Home Assistant
 device. See [Key expiry][tailscale_info_key_expiry] for more information.
 
+To allow Home Assistant and other add-ons to access your Tailscale network the
+add-on also provides a SOCKS5/HTTP proxy. The proxy protocol is unauthenticated.
+See [Userspace networking][tailscale_info_userspace_networking] for more
+information.
```

Comment:
This is actually not usable as such. One won't be able to leverage the socks proxy for the described services.